tests/framework/installer/ceph_manifests.go

```diff
@@ -81,7 +82,17 @@ func (m *CephManifestsMaster) Settings() *TestCephSettings {
 	return m.settings
 }
 
-func (m *CephManifestsMaster) GetCRDs(k8shelper *utils.K8sHelper) string {
+func (m *CephManifestsMaster) GetCRDs(t *testing.T, k8shelper *utils.K8sHelper) string {
+	// k8sVersion, err := k8sutil.GetK8SVersion(k8shelper.Clientset)
+	// if err != nil {
+	// 	assert.NoError(t, err)
+	// }
+	// minValidatinPolicy := k8sVersion.AtLeast(version.MustParseSemantic(minValidatinPolicy))
+	// // For multi-cluster integration we create a internal mode cluster and external mode cluster both
+	// // but since CRDs are not namespaced tight we need to use pre-1.25/crds.yaml so that CI works.
+	// if !minValidatinPolicy {
+	// 	return m.settings.readManifest("pre-1.25/crds.yaml")
```

Comment:
Great to see the CI is working now without a different version of the CRDs! So we can revert the pre-1.25 crds.yaml, right?

Reply:
reverted